projects/ngx-datatable/src/lib/components/datatable.component.ts

```diff
@@ -370,17 +370,7 @@ export class DatatableComponent<TRow = any>
    * totalMessage     [default] = 'total'
    * selectedMessage  [default] = 'selected'
    */
-  @Input() messages: Partial<INgxDatatableConfig['messages']> = {
-    // Message to show when array is presented
-    // but contains no values
-    emptyMessage: 'No data to display',
-
-    // Footer total message
-    totalMessage: 'total',
```

Comment:
Don't we miss the default for `totalMessage` in the template?

<img width="545" alt="image" src="https://github.com/user-attachments/assets/3781d231-a874-4dde-8820-7ada9c54b200" />

Reply:
Originality moved into footer.component.ts:
![image](https://github.com/user-attachments/assets/11dd4083-5caa-49bc-a3e8-f07532c8b579)

But I changed it again to be in the datatable.component.html, so we can make totalMessage non-nullable in the footer component once we enabled strict Null checks.